README.md

```diff
@@ -78,3 +78,79 @@ To build the full distribution, with all database bindings:
 To build a single database binding:
 
     mvn -pl site.ycsb:mongodb-binding -am clean package
+
+Running multiple instances and latency percentiles
+--------------------------------------------------
+
+In general, you shall be interested in 99% percentile (P99) of the latency
+distribution and the rest - 99.9%, 99.99%, 99.999% - the latency tail.
+95% percentile has a very low probability to be observed by the user
+so don't even bother (see "How Many Nines?" at
+https://bravenewgeek.com/everything-you-know-about-latency-is-wrong/
+). The formula to calculate probability of how many clients will
+observe a specific percentile is: 
+
+    Probability_to_observe = 1 - Percentile ^ Requests
+
+That is why almost 30% of the users will observe P99 bad latency just by 
+loading the default _google.com_ web page:
+
+    1 - 0.99 ^ 30 = 0.27
+
+Remember, that
+
+- _latencies_ percentiles can't be averaged. Don't fall into this
+  [trap](http://latencytipoftheday.blogspot.com/2014/06/latencytipoftheday-you-cant-average.html).
+  Neither latency averages, nor P99 averages do not make any sense.
+
+If you run multiple loaders dump result histograms with:
+
+    -p hdrhistogram.fileoutput=true
+    -p hdrhistogram.output.path=file.hdr
+
+merge them manually and extract required percentiles out of the
+joined result.
+
+Remember that running multiple workloads may distort original
+workloads distributions they were intended to produce.
+
+Merging HDR histogram percentiles
```

Comment:
I like that you pointed the user to tool to help them but I think it would be better to link them to the documentation for that tool rather than providing a synopsis here. As the HDRLogProcessing tool changes the likelihood of this documentation being updated is fairly low which will lead to questions and confusion for users.

Reply:
I removed all examples and left the link only to the tool github which has a README. I left an example for CSV extraction. If you want ping me to remove it.